neo/Storage/Implementation/LevelDB/LevelDBClassMethods.py

```diff
@@ -0,0 +1,235 @@
+import plyvel
+import threading
+
+from contextlib import contextmanager
+
+from neo.Core.Blockchain import Blockchain
+from neo.Storage.Common.DBPrefix import DBPrefix
+from neo.Storage.Interface.DBInterface import DBProperties
+from neo.logging import log_manager
+
+
+logger = log_manager.getLogger()
+
+"""
+Description:
+    All the methods are used within the dynamically generated class from the
+    database factory.
+Usage:
+    For a new database implementation all methods defined in
+    AbstractDBImplementation have to be implemented.
+"""
+
+# name of the init method for a generic database class
+_init_method = '_db_init'
+
+# name of the init method for a prefixed database class
+_prefix_init_method = '_prefix_db_init'
+
+# path where the data files are stored
+_path = None
+
+# either the real database class, in this case a LevelDB instance,
+# or a prefixed db or snapshot
+_db = None
+
+# iterator instance created within openIter(self, properties)
+_iter = None
+
+# batch instance to perform batch operations on the database
+_batch = None
+
+_lock = threading.RLock()
+
+
+@property
+def Path(self):
```

Comment:
is this a left over? Couldn't find any references to it

Reply:
Yes I thought it's needed at first, removed now.